Dockerfile.bundle

```diff
@@ -1,12 +1,19 @@
 FROM scratch
 
+# Define build arguments to receive dynamic values
+ARG short-commit
+ARG git-url
```

Comment:
I think that we can skip this part from the "community" Dockerfile. This will add unnecessary complexity for local builds.